target/linux/bcm53xx/base-files/etc/board.d/01_leds

```diff
@@ -5,6 +5,15 @@
 board_config_update
 
 case "$(board_name)" in
+linksys,panamera)
+	ucidef_set_led_usbport "usb2" "USB 2.0" "bcm53xx:green:usb2" "usb1-port2" "usb2-port2"
+	ucidef_set_led_usbport "usb3" "USB 3.0" "bcm53xx:green:usb3" "usb1-port1" "usb2-port1" "usb4-port1"
+	ucidef_set_led_netdev "wifi-enabled" "Wireless5G1" "bcm53xx:white:wifi-enabled" "wlan0" "link"
+	ucidef_set_led_default "wifi-disable-button" "Wifi-Disable" "bcm53xx:amber:wifi-disabled" "1"
+	ucidef_set_led_default "wps-disable-button" "Wifi-Disable" "bcm53xx:white:wps" "1"
+	ucidef_set_led_default "logo" "Power" "bcm53xx:white:power" "1"
```

Comment:
Do we really need all these boolean LEDs defined here?

Reply:
It's a typo, I'll fix it

Reply:
These are necessary

Reply:
This somewhat looks like you are abusing LED definitions for setting GPIOs? Or how does a LED "wifi-disable" make sense at all?

Reply:
There is a group of eight white-led vertical bars on the router's face which will turn off when wifi is off according to the manual. These cannot be controlled individually:

![image](https://user-images.githubusercontent.com/26283118/108723720-94114700-74f2-11eb-8383-eef7e971e67c.png)

At the moment, the MMIO leds cannot be programmed as factory. I've left them out.

Reply:
Still, 01_leds is for defining LED _triggers_. Buttons or GPIO switches to enable anything (including LEDs) do not belong here.
Typical ways to implement that would be stuff like gpio-export, gpio hogs, or user-space code in a separate file like here:
https://github.com/openwrt/openwrt/blob/master/target/linux/ath79/generic/base-files/etc/board.d/03_gpio_switches

While I have no idea what's the proper way to handle this on bcm53xx (probably @rmilecki can help here), just abusing LED functions for buttons/switches should not be the way to go.

Reply:
I clubbed them here only because it pertains to LEDs.  If needed I can certainly  move "Wifi-Disable" and "WPS-Disable" button to separate file. I just need a confirmation from you or Rafal to do so.

Reply:
Removed them for now, I'll submit a separate pull request once I get a confirmation. I'll include proper explanation for those. Thanks